bin/enable_printfs

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
```

Comment:
Also, probably worth making this explicitly `python3`, but glancing through it it looks like it's compatible with both.

Reply:
compatibility the goal